src/css/components/_control.scss

```diff
@@ -36,3 +36,28 @@
   display: table-cell;
   vertical-align: middle;
 }
+
+// visually hide a control-bar component without changing the position
+// and without hiding it from screen readers. This also preserves
+// a transition animation.
+.video-js .vjs-control .vjs-visual-hide-vertical {
```

Comment:
Ok, I see what there are two. I still wonder if there's some way to simplify it.

Also, I just noticed that when the volume control is hidden as part of the volume panel, it adds a pixel to the width of the panel, we probably don't want that. So, we could potentially use the webaim hiding technique on a delay after the height/width transition?

Also, I noticed a small hiccup with the animation:
![volume control animating as part of volume panel](https://cldup.com/Q1H2xC2ng3.gif)
Notice the small white bar that shows up at of the control closing. This potentially stems from the same issue as the last paragraph.